lib/ebpd_loader.h

```diff
@@ -0,0 +1,14 @@
+#ifndef __EBPD_LOADER_H_
+#define __EBPD_LOADER_H_
+
+class EbpdLoader {
+    public:
+        EbpdLoader() { }
+        ~EbpdLoader();
+        int LoadFile(const char *filepath, int ifindex);
+        int LoadBuffer(void *buf, int buf_size, const char *name);
```

Comment:
uhm .. why LoadFile requires an ifindex, an interface to which to attach the eBPF program, but LoadBuffer does not?

name is a 'string name' for the buffer? I assume it is not an interface name? I would document parameters in a comment.

Reply:
This is a potential problem with bpf_object__open_buffer() API which doesn't take ifindex as argument. We will have to check with kernel guys and get some dump on how to use these APIs. The string name is just a user given name.